libs/gi/page-team/src/TeamCharacterSelector.tsx

```diff
@@ -94,7 +108,11 @@ export default function TeamCharacterSelector({
           </Typography>
         </CardContent>
       </BootstrapTooltip>
-      <Tabs variant="fullWidth" value={characterKey ?? 'team'}>
+      <Tabs
+        variant="fullWidth"
```

Comment:
Why not scrollable for mobile as well? Too janky?

Reply:
Janky cause the background colors does not follow scroll (and its more work than its worth to make them follow)